04_environment/index.tex

```diff
@@ -20,41 +20,32 @@ \subsubsection{Modifications to the Original concepts}
 
-The benefit of deer hunting is higher payoffs. However, these improved returns come at the cost of significantly higher tier thresholds and a “zero” tier. Therefore, it is possible for a single island to go deer hunting without enough resources to reach the hunting location leading to no return.
+The focus of fishing is to avoid risk at the expense of payoff. This means that all tier requirements are lower than the deers. However, the return is also lower thus if only one island goes fishing they are still expected to have a low return as long as they have enough resources to catch the first fish. The benefit of deer hunting is higher payoffs. However, these improved returns come at the cost of significantly higher tier thresholds. Therefore, it is possible for a single island to go deer hunting without enough resources to reach the hunting location leading to no return.
 
 \begin{equation}
-Increments\ of\ catching\ n\ deer's=
+\text{Increments of catching $n$ animals}=
 \left( \begin{array}{ll@{}}
-n=0 -> \Delta^0 = 1 \\
-n=1 -> \Delta^1 = 0.8^1 \\
-n=2 -> \Delta^2 = 0.8^2 = 0.64 \\
-n=2 -> \Delta^3 = 0.8^3 = 0.512 \\
-n=4 -> \Delta^4 = 0.8^4 = 0.4096\\
+n=0 \longrightarrow \Delta^0 = 1 \\
+n=1 \longrightarrow \Delta^1 = 0.8^1 \\
+n=2 \longrightarrow \Delta^2 = 0.8^2 = 0.64 \\
+n=2 \longrightarrow \Delta^3 = 0.8^3 = 0.512 \\
+n=4 \longrightarrow \Delta^4 = 0.8^4 = 0.4096\\
 \end{array} \right) 
 \label{eq:Cumulative Cost 1}
 \end{equation}
 
 \begin{equation}
-Cumulative\ deer \ cost \sum_{n=0}^{n} \Delta^{n} = \Delta^{0} + \Delta^{1} + \Delta^{2} + \Delta^{3} + \Delta^{4} + ....
+\text{Utility Tier Cost}=\ \sum_{n=0}^{n} \Delta^{n} = \Delta^{0} + \Delta^{1} + \Delta^{2} + \Delta^{3} + \Delta^{4} + ....
 \label{eq:Cumulative Cost 2}
 \end{equation}
 
-\begin{equation}
-Cumulative\ fishing\ cost \ = \ \sum_{n=1}^{n} \Delta^{n} = \Delta^{1} + \Delta^{2} + \Delta^{3} + \Delta^{4} + \Delta^{5} + ....
-\label{eq:Cumulative Cost 3}
-\end{equation}
-
-Equations that show the formulation of the decay function for tier system.
-
-The tier system is shown in Equation~\eqref{eq:Cumulative Cost 1}, Equation~\eqref{eq:Cumulative Cost 2} and Equation~\eqref{eq:Cumulative Cost 3}, where $n$ stands for the tiers, with decay costs ($\Delta$) of $0.8$ for both the deer hunt and fishing. Fishing cost starts at $n$ is equal to $1$ due to the reduced cost.
+Equation~\eqref{eq:Cumulative Cost 1} and Equation~\eqref{eq:Cumulative Cost 2} show the formulation of the decay function for tier system. The tier system is shown in the ~\ref{fig:Foraging Tier System} where $n$ stands for the tiers, with decay costs ($\Delta$) of $0.8$ for the deer hunt and $0.6$ for fishing.
```

Comment:
good label for equations

The tier system is shown in the **Figure**~\ref{fig:Foraging Tier System}